NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -91,12 +94,8 @@ internal Optional<World> LoadFromFile(string saveDir)
             try
             {
                 PersistedWorldData persistedData = new();
-                SaveFileVersion saveFileVersion = Serializer.Deserialize<SaveFileVersion>(Path.Combine(saveDir, $"Version{FileEnding}"));
 
-                if (saveFileVersion == null || saveFileVersion.Version != NitroxEnvironment.Version)
-                {
-                    throw new InvalidDataException("Version file is empty or save data files are too old");
-                }
```

Comment:
I think those kind of data are good to keep, maybe just a log

Reply:
Yeah the `saveFileVersion == null` check should be kept imo.

Reply:
Wont this get caught by the check of is the file version.json exists? I believe the removed saveFileVersion with deserialization defaults to version 1.5.0.0 when it cannot deserialise the content so should never be null

Reply:
Even if the file does not exist or is corrupted?

Reply:
If it doesnt exist then the [check](https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxServer/Serialization/World/WorldPersistence.cs#L85) for a previous file should catch it unless I am mistaken. If it is currupted like for example a 1.4 file with the BaseDataVersion then it defaults to 1.5.0.0